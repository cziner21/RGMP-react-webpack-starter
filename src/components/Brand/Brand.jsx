import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.div`
    color: #f65261;
`

function Brand() {
    return (
        <Wrapper>
            <strong>netflix</strong>roulette
        </Wrapper>
    )
}

export default Brand
