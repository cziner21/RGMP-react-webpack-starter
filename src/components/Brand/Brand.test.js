import React from 'react'
import renderer from 'react-test-renderer'

import Brand from './Brand.jsx'

describe('Brand component test', () => {
    it('should renders correctly', () => {
        const brand = renderer.create(<Brand />).toJSON()
        expect(brand).toMatchSnapshot()
    })
})
