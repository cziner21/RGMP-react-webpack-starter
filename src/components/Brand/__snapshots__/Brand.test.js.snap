// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand component test should contain text "netflix" 1`] = `
<div
  className="sc-bczRLJ jQKpnh"
>
  <strong>
    netflix
  </strong>
  roulette
</div>
`;

exports[`Brand component test should renders correctly 1`] = `
<div
  className="sc-bczRLJ jQKpnh"
>
  <strong>
    netflix
  </strong>
  roulette
</div>
`;
